packages/titanium-data-table/src/titanium-data-table.ts

```diff
@@ -20,9 +18,10 @@ declare const ResizeObserver: any;
  * @element titanium-data-table
  *
  * @fires selected-changed - Fired when a row or rows in the data table is selected. detail: array<unknown>
- * @fires page-changed - Fired when item is selected.  detail: { isSelected: boolean, item: unknown }
- * @fires take-changed - Fired when item is selected.  detail: { isSelected: boolean, item: unknown }
+ * @fires page-changed - DEPRECATED Use `paging-changed` instead.
```

Comment:
This isn't deprecated, it's removed right?

Reply:
yeah true. It is removed but maybe would still want it in the docs? Maybe I could add a deprecated as of like a specific version number?